charts/airbyte/values.yaml

```diff
@@ -1319,10 +1325,10 @@ minio:
     tag: RELEASE.2023-11-20T22-40-07Z
 
   auth:
-    rootUser: minio
-    rootPassword: minio123
+    rootUser: ""
+    rootPassword: ""
   storage:
-    volumeClaimValue: 500Mi
+    volumeClaimValue: 5Gi
```

Comment:
Does the volume need to have that much space or does this create a volume with that much space?

Reply:
This is the amount of storage space requested from the k8s cluster's storage system. We need this much space along with a TTL policy in place for the storage bucket because Airbyte produces a lot of logs, and jobs start failing once the storage is full.